### DISCUSSION TITLE
How to disable showing selected elements on dropdown menu

### DISCUSSION THREAD

**Discussion by sahilahluwalia** [dowjones/react-dropdown-tree-select#626] [Q&A] [answered]
How to disable the creation of box elements on dropmenu itself, which i have selected in dropmenu. i want to have a clean look like dropmenu have when no elements are selected. how can i do that?

![image](https://user-images.githubusercontent.com/44433679/235138987-647e9722-9f5d-40ff-9a44-bd5b74415fd9.png)
This is the clean look when no elements is selected. i wanted this same look even entries are selected in dropmenu

**Comment by mrchief** [comment, marked as answer]
You mean not to show that tags at all? You can just add a bit of css to hide them.

**Reply by sahilahluwalia** [reply]
thanks!
